pkg/kubelet/kubelet.go

```diff
@@ -1458,6 +1458,13 @@ func (kl *Kubelet) syncPod(pod *api.Pod, mirrorPod *api.Pod, runningPod kubecont
 		return err
 	}
 
+	if kubepod.IsStaticPod(pod) && mirrorPod == nil {
```

Comment:
Forgot to update `mirrorPod` with the fresh one. Now the block at 1567 will update the mirror.